src/commands/cloud-login.js

```diff
@@ -1,41 +1,54 @@
 const {cli} = require('cli-ux')
-const {flags} = require('@oclif/command')
 const {validCloudEndpoint, saveEndpointOrError, errorOut} = require('../lib/misc.js')
 const FaunaCommand = require('../lib/fauna-command.js')
 const os = require('os')
-const rp = require('request-promise')
+const fetch = require('request-promise')
 const url = require('url')
-const http = require('http')
-const crypto = require('crypto')
-const childProcess = require('child_process')
 
 const SHELL_LOGIN_URL = 'https://app.fauna.com/shell_login'
-const GITHUB_HTTP_HANDLER_PORT = 8086
-const GITHUB_CLIENT_ID = '615935a34846151ad806'
 const alias = 'cloud'
 const CLOUD_URL = 'https://db.fauna.com'
 const newEndpoint = url.parse(CLOUD_URL)
+const EMAIL_STRATEGY = 'email'
+const SECRET_STRATEGY = 'secret'
+const strategies = {
+  [EMAIL_STRATEGY]: emailStrategy,
+  [SECRET_STRATEGY]: secretStrategy,
+}
 
 class CloudLoginCommand extends FaunaCommand {
   async run() {
-    const {flags} = this.parse(CloudLoginCommand)
-
     return validCloudEndpoint()
-    .then(async function (_) {
-      if (flags.github) {
-        return githubStrategy(flags.no_browser)
-      } else {
-        return cloudStrategy()
-      }
-    })
+    .then(() => cloudStrategy({log: this.log}))
     .catch(function (err) {
       errorOut(err.message, 1)
     })
   }
 }
 
-async function cloudStrategy() {
-  const email = await cli.prompt('Email', {timeout: 120000})
+async function cloudStrategy({log}) {
+  log('For email login, enter your email below, and then your password.')
+  log('For login from with 3rd-party identity providers like Github or Netlify, please acquire a key from Dashboard > Security and enter it below instead.')
```

Comment:
"for login from with" -> "for login with"

Reply:
Fix this text and we're good to merge